proteobench/modules/quant/quant_lfq_ion_DDA_QExactive.py

```diff
@@ -113,71 +114,14 @@ def benchmarking(
-                f"Error parsing {input_format} file, please make sure the format is correct and the correct software tool is chosen: {e}"
-            ) from e
-        except Exception as e:
-            raise ParseSettingsError("Error parsing the input file.") from e
-
-        msg = f"Folder: {self.parse_settings_dir}, Module: {self.module_id}"
-        # Parse settings file
-        try:
-            parse_settings = ParseSettingsBuilder(
-                parse_settings_dir=self.parse_settings_dir, module_id=self.module_id
-            ).build_parser(input_format)
-        except KeyError as e:
-            raise ParseSettingsError(
-                f"Error parsing settings file for parsing, settings seem to be missing: {msg}"
-            ) from e
-        except FileNotFoundError as e:
-            raise ParseSettingsError(f"Could not find the parsing settings file: {msg}") from e
-        except Exception as e:
-            raise ParseSettingsError(f"Error parsing settings file for parsing: {msg}") from e
-
-        try:
-            standard_format, replicate_to_raw = parse_settings.convert_to_standard_format(input_df)
-        except KeyError as e:
-            raise ConvertStandardFormatError("Error converting to standard format, key missing.") from e
-        except Exception as e:
-            raise ConvertStandardFormatError("Error converting to standard format.") from e
-
-        # instantiate quantification scores
-        try:
-            quant_score = QuantScores(
-                self.precursor_column_name, parse_settings.species_expected_ratio(), parse_settings.species_dict()
-            )
-        except Exception as e:
-            raise QuantificationError("Error generating quantification scores.") from e
-
-        # generate intermediate data structure
-        try:
-            intermediate_metric_structure = quant_score.generate_intermediate(standard_format, replicate_to_raw)
-        except Exception as e:
-            raise IntermediateFormatGenerationError("Error generating intermediate data structure.") from e
-
-        # try:
-        current_datapoint = QuantDatapoint.generate_datapoint(
-            intermediate_metric_structure, input_format, user_input, default_cutoff_min_prec=default_cutoff_min_prec
-        )
-        # except Exception as e:
-        #    raise DatapointGenerationError(f"Error generating datapoint: {e}")
-
-        # add current data point to all datapoints
-        # try:
-        all_datapoints = self.add_current_data_point(current_datapoint, all_datapoints=all_datapoints)
-        # except Exception as e:
-        #    raise DatapointAppendError(f"Error adding current data point: {e}")
-
-        # return intermediate data structure, all datapoints, and input DataFrame
-        # TODO check why there are NA and inf/-inf values
-        return (
-            intermediate_metric_structure,
-            all_datapoints,
-            input_df,
+        return run_benchmarking(
```

Comment:
ah so here it replaces the function. ok. ignore my previous comment regarding the webinterface